.circleci/config.yml

```
@@ -283,6 +294,28 @@ jobs:
             - dist-flask
             - builds-flask
 
+  prep-build-test-flask:
+    executor: node-browsers-medium-plus
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Build extension for testing
+          command: yarn build:test:flask
+      - run:
+          name: Move test build to 'dist-test' to avoid conflict with production build
+          command: mv ./dist ./dist-test-flask
+      - run:
+          name: Move test zips to 'builds-test' to avoid conflict with production build
```

Nit: Updating directory name in comment

```suggestion
          name: Move test build to 'dist-test-flask' to avoid conflict with production build
          command: mv ./dist ./dist-test-flask
      - run:
          name: Move test zips to 'builds-test-flask' to avoid conflict with production build
```